spec/services/sign_in/session_refresher_spec.rb

```diff
@@ -132,25 +132,33 @@
                 let(:refresh_token_duration) { SignIn::Constants::RefreshToken::VALIDITY_LENGTH_SHORT_MINUTES }
                 let(:updated_session_expiration) { Time.zone.now + refresh_token_duration }
 
+                # This example has a failure due to precision
+                # expected `SignIn::OAuthSession#refresh_expiration` to have
+                # initially been:
+                # <ActiveSupport::TimeWithZone 2024-01-04 01:23:09+(439437107/1000000000) +00:00 (UTC)>,
+                # but was
+                # <ActiveSupport::TimeWithZone 2024-01-04 01:23:09+(439437/1000000) +00:00 (UTC)>
+                # be_within((0.1).second fixes the precision issue
                 it 'updates the session with a new expiration time' do
                   expect do
                     subject
                     session.reload
-                  end.to change(session, :refresh_expiration).from(session_expiration)
-                                                             .to(updated_session_expiration)
+                  end.to change(session, :refresh_expiration).from(be_within(0.1.seconds).of(session_expiration))
```

Comment:
Same here, would you be able to round the time expectations in the `session_expiration`/`updated_session_expiration` definitions themselves?